packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -40,10 +40,10 @@ class ProgressivePaginator extends React.Component {
   constructor(props) {
     super(props);
 
-    const { selectedPage, totalCount } = this.props;
+    const { selectedPage } = this.props;
 
     this.state = {
-      selectedPage: selectedPage && totalCount ? selectedPage : undefined,
+      selectedPage: selectedPage || 1,
```

Comment:
To maintain existing logic, should this be
```suggestion
selectedPage: selectedPage && totalCount ? selectedPage : 1,
```

Reply:
when `totalCount` is not provided then `selectedPage` value will also gets ignored in this case. (Since totalcount is optional prop now there may be scenario where we still need to consider selectedPage value in progressive paginator)